examples/resources/awscc_logs_destination/cloudwatch_log_destination_with_dest_policy.tf

```diff
@@ -0,0 +1,81 @@
+data "aws_region" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+
+resource "awscc_kinesis_stream" "this" {
+  name                   = "terraform-kinesis-test"
+  retention_period_hours = 48
+  shard_count            = 1
+  stream_mode_details = {
+    stream_mode = "PROVISIONED"
+  }
+}
+
+data "aws_iam_policy_document" "cloudwatch_assume_role_policy" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["logs.amazonaws.com"]
+    }
+  }
+}
+
+locals {
+  destination_name = "test_destination"
+}
+
+resource "awscc_iam_role" "main" {
+  role_name                   = "sample_iam_role"
+  description                 = "This is a sample IAM role"
+  assume_role_policy_document = data.aws_iam_policy_document.cloudwatch_assume_role_policy.json
+  managed_policy_arns         = ["arn:aws:iam::aws:policy/AmazonKinesisFullAccess"]
+  path                        = "/"
+  tags = [
+    {
+      key   = "Name"
+      value = "Sample IAM Role"
+    },
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
```

Comment:
same thing here, I feel that this tags is a distraction and did not serve additional purposes for the reader